fdbclient/AuditUtils.actor.cpp

```diff
@@ -142,22 +146,29 @@ ACTOR Future<Void> persistAuditState(Database cx,
                                      MoveKeyLockInfo lock,
                                      bool ddEnabled) {
 	state Transaction tr(cx);
+	state AuditPhase auditPhase = auditState.getPhase();
 
 	loop {
 		try {
 			tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
 			tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
-			auto auditPhase = auditState.getPhase();
-			if (auditPhase == AuditPhase::Complete || auditPhase == AuditPhase::Failed ||
-			    auditPhase == AuditPhase::Error) {
-				wait(checkMoveKeysLock(&tr, lock, ddEnabled, true));
-			}
-
+			ASSERT(auditPhase == AuditPhase::Complete || auditPhase == AuditPhase::Failed ||
```

Comment:
nit: Move this ASSERT to the beginning of the function?

Reply:
Moved